projects/transactions-journey/src/lib/components/transaction-item/transaction-item.component.html

```diff
@@ -0,0 +1,15 @@
+<span class="categoryCode" [ngStyle]="{'background-color': transaction.categoryCode}"></span>
+<span class="date">{{ transaction.dates.valueDate | date: 'MMM. d' }}</span>
+<span class="description">
+  <span class="merchant">{{ transaction.merchant.name }}</span>
```

Comment:
Shall we follow BEM methodology for naming classes here as proposed by Design System team? Or it is not necessary?

Reply:
done